symplyphysics/laws/relativistic/vector/relativistic_velocity_parallel_to_movement.py

```diff
@@ -0,0 +1,70 @@
+from sympy.physics.units import speed_of_light
+from symplyphysics import (
+    units,
+    Quantity,
+    validate_input,
+    validate_output,
+    Vector,
+    dot_vectors,
+    add_cartesian_vectors,
+    scale_vector,
+    QuantityVector,
+    cross_cartesian_vectors,
+    vector_magnitude,
+)
+
+# Description
+## Consider two inertial reference frames: one fixed (lab frame) and one tied to the moving object (proper frame).
+## The proper frame is moving with some velocity `v` relative to lab frame. Then, according to the theory of special
+## relativity, the velocity of the object relative to lab frame is not equal to the sum of its velocity in the proper
+## frame and the velocity of the proper frame relative to the lab frame.
+
+# Law: u_parallel = (u_parallel' + v) / (1 + dot(u_parallel', v) / c**2)
+## u_parallel - velocity vector relative to lab frame in the perpendicular direction
+## u_parallel' - velocity vector relative to proper frame in the perpendicular direction
+## v - velocity vector of proper frame relative to lab frame
+## c - speed of light
+
+
+def parallel_addition_law(
```

Comment:
Rename to make clear, what the result of addition is.

Reply:
Maybe just name such functions just `law`? The content of the law should evident from the module name

Reply:
Functions are usually named as verbs, i.e. `it_does_something()`. And if we are looking to get `u_parallel'` or `v` from this law, we would need 2 additional functions.

Reply:
I just thought that this is somewhat inconsistent with the fact that we have `law`s in scalar laws.

We can reuse the same function to find `u_paralell'`. `proper` and `lab` frames switch, the only difference is that `v` becomes `-v`. I don't really think we need a separate function for that, it's redundant from the physical standpoint.

And I'm not sure there is a formula for `v`.

Reply:
Yes, we could do it. But with `solve` we could get solution for `u_parallel'` automatically and fool proof. It is easy to make a mistake when substituting it manually. Also requires to thoroughly read the description, or know in advance, how to make the translation.

Reply:
Added them